.github/workflows/test.yml

```
@@ -293,3 +329,12 @@ jobs:
             ${{ steps.reports.outputs.path }}/${{ matrix.name }}.xml
             ${{ steps.reports.outputs.path }}/${{ matrix.name }}.json
         continue-on-error: true
+
+      - name: Create issue on failure
+        if: failure() && github.event_name == 'schedule'
+        uses: ipdxco/create-or-update-issue@0265e9148d6c26f6cdb0abb9c47df048a585c2fe # v1.0.0
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          title: "Fix Expensive Tests"
+          label: "expensive-test-failure"
```

```suggestion
          label: "area/expensive-test-failure"
```

I created this label.

Looking at https://github.com/ipdxco/create-or-update-issue/blob/main/index.js#L19-L26, it seems like we don't do any sorting of issues that have this label and then we take the first in the response.  Should we maybe order by creation date so the order is deterministic and take the newest?  

I'm thinking about the workflow for when an issue starts accumulating a lot of failures.  At some point, the maintianers may want to "fork" and have a new issue start accumulating failures.  To do that, they could either:
1. Remove the "area/expensive-test-failure" label OR
2. Create a new issue and assign the new issue the "area/expensive-test-failure" label.
With the proposal to grab the newest "area/expensive-test-failure" labeled issue, either approach will cause the old issue to no longer get updates, which seems good/reasonable.

---

That's a good usecase, I release a v1.0.1 to make the ordering explicit (sorting by descending creation date is the default)